sdk/identity/azure-identity/test/ut/token_credential_impl_test.cpp

```diff
@@ -872,7 +872,7 @@ TEST(TokenCredentialImpl, Diagnosability)
           "}",
           "TokenForAccessing",
           "TokenExpiresInSeconds",
-          {}));
+          ""));
```

Comment:
Why are we changing this?

Reply:
`{}` could mean "empty vector" and "empty string". The new syntax unambiguously specifies it as "empty string".

Reply:
I updated the code to use `{}`, yet still not be ambiguous about its meaning (it won't compile otherwise) due to ambiguity.